src/app/feature/consultant/components/consultant-picture-edit/consultant-picture-edit.component.ts

```diff
@@ -48,9 +50,15 @@ export class ConsultantPictureEditComponent implements OnDestroy {
         urlProfileImage: this.croppedImage
       };
 
-      this.consultantStore.updateConsultant(updatedData)
-        .pipe(takeUntil(this.destroy$))
-        .subscribe(() => this.close());
+      if(this.croppedImage){
+        this.consultantStore.updateConsultant(updatedData)
+           .pipe(takeUntil(this.destroy$))
```

Comment:
Instead of checking if this.croppedImage is truthy, you could add a catchError rxjs operator into the pipe which could call the error message snackbar.